test/run-drun/stable-region-migration/actor1.mo

```diff
@@ -0,0 +1,33 @@
+//MOC-FLAG --stable-regions
+
+import P "mo:⛔";
+import M "../stable-mem/StableMemory";
+import Region "../stable-region/Region";
+
+actor {
+
+    // measure out three blocks' worth of bytes.
+    let pageInBytes = 1 << 16 : Nat64;
+    let blockInBytes = pageInBytes * 128 : Nat64;
+    let size = blockInBytes * 3 : Nat64;
+    var i = 0 : Nat64;
+
+    // Check size for necessary number of pages.
+    let reqPages = size / pageInBytes;
+    assert M.size() == reqPages;
+
+    // Load out previously-stored byte pattern, one byte at a time.
+    // Check each byte is what we would have written, if we were repeating the same logic again.
+    while (i < size) {
+        let expected = P.natToNat8(P.nat64ToNat(i % 256)) : Nat8;
+        let loaded = M.loadNat8(i);
+        assert loaded == expected;
+        i := i + 1;
+    };
+
+    P.debugPrint ("actor1: checked region0.");
+
+    stable var r1 = Region.new();
```

Comment:
@crusso I'm a bit stuck here, in terms of testing.

This is the test I want, but I was doing some sanity checking and found out that
the `MOC-FLAG --stable-regions` comment is being ignored by the `.drun` script.

It doesn't prevent me from completing the migration logic, but it prevents me from testing it the way that I wanted.

Reply:
See the current test output, which shows two things:
- The check actually succeeds.  And, since we are missing the migration logic still (just a stub in the place I think it needs to go), that means we are running the old API, as the new API would barf without proper meta data being set up for the region manager.
- To verify that, I made it so `Region.new` checks for the flag, and traps without it.  [See the test output to see that indeed, that trap arises.](https://github.com/dfinity/motoko/pull/3768/files#r1239008645)